internal/apis/acme/types_issuer.go

```diff
@@ -102,6 +102,12 @@ type ACMEIssuer struct {
 	// it, it will create an error on the Order.
 	// Defaults to false.
 	EnableDurationFeature bool
+
+	// Sets the overall timeout for creating certificates in seconds. For most
+	// Issuers the default should be sufficient, however e.g. letsencrypt
+	// staging sometimes takes longer, especially on slow local environments.
+	// Defaults to 20s
+	AuthorizationTimeout string
```

Comment:
Is there a metav1.Duration type we can use here instead?

Reply:
good point, I'll change it (hopefully this weekend) and force-push.

Reply:
solved with 7da7daa3d9aacdd20d009e1698e15e77fb390ecc